src/WebDriver.php

```diff
@@ -266,6 +266,7 @@ public function start()
                 $this->applyTimeouts();
             }
             $this->rootWindow = $this->webDriver->getWindowHandle();
+            $this->windows = [];
```

Comment:
This doesn’t affect reset, because reset keeps the same session, and the window handles do not reset in the browser so the id is never reused.

This issue only affects new sessions because the browser window handles start from the first number again.

We restart the session between each test for better test isolation so we only call restart or stop/start. We never call reset because we don’t need to.